easybuild/easyconfigs/d/DIAMOND/DIAMOND-2.0.4-GCC-9.3.0.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'DIAMOND'
+version = "2.0.4"
+
+homepage = 'https://github.com/bbuchfink/diamond'
+description = "Accelerated BLAST compatible local sequence aligner"
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
+
+source_urls = ['https://github.com/bbuchfink/%(namelower)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['94e8fe72bdc28b83fd0f2d90c439b58b63b38263aa1a3905582ef68f614ae95d']
+
+separate_build_dir = True
+
+builddependencies = [
+    ('binutils', '2.34'),
```

Comment:
Same remarks as in #11682: please drop `binutils` build dep + add (back) missing `zlib` dependency